autoprotocol/constants.py

```diff
@@ -35,3 +35,5 @@
     (0.6243159222654036, -0.14191423819668741),
     (0.563207308766221, 0.5152506858187523),
 ]
+
+PROVISION_MEASUREMENT_MODES = ["mass", "volume"]
```

Comment:
nit: could we move this into the `Provision` Instruction? or the protocol method?

Reply:
As they are really serving as constants and we have separate file for that, I feel it is the right place.